client/app/models/package.js

```diff
@@ -0,0 +1,18 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class PackageModel extends Model {
+  @belongsTo('project')
+  project;
+
+  @belongsTo('pas-form')
```

Comment:
Should this relationship be that `package belongsTo pas-form`, or `package hasOne pas-form`?

Reply:
Thanks! You're right, logically it would be a hasOne. But you could also say the same for a pas-form: it has one package. There's a 1-to-1 relationship between the two.

The `hasMany` and `belongsTo` verbiage is borrowed from Rails' ActiveRecord Association API, which was designed for abstracting how relational databases work. In the case of relational databases, this makes sense: there's a real difference between belongsTo and hasOne because one expects a foreign key, the other infers it. 

However, in Ember Data, these APIs were not designed for relational databases, or any specific database in general. It's just linking together the in-memory relationships between JSON objects, which may be by their nature nested and bi-directional.

All that to say, there is no `hasOne` in Ember Data — in this case, `belongsTo` does the same thing, and a 1-to-1 relationship should use `belongsTo` on both sides.

Reply:
Awesome, thanks for clarifying and explaining that @allthesignals, totally makes sense :-)